dev/create-release/translate-contributors.py

```diff
@@ -31,7 +31,10 @@
 import os
 import sys
 
-from releaseutils import *
+import unidecode
```

Comment:
I think we should do:

```python
try:
    import unidecode
except ImportError:
    print("This tool requires the unidecode library to decode obscure github usernames")
    print("Install using 'sudo pip install unidecode'")
    sys.exit(-1)
```

to keep the same behaviour for now.

Reply:
Good one! Updated it in the PR :)